arch/arm/boot/dts/overlays/ov64a40.dtsi

```diff
@@ -0,0 +1,33 @@
+// Fragment that configures an OV64A40
+
+cam_node: ov64a40@36 {
+	compatible = "ovti,ov64a40";
+	reg = <0x36>;
+	status = "disabled";
+
+	clocks = <&cam1_clk>;
+	clock-names = "xclk";
+
+	avdd-supply = <&cam1_reg>;	/* 2.8v */
+	dovdd-supply = <&cam_dummy_reg>;/* 1.8v */
+	dvdd-supply = <&cam_dummy_reg>;	/* 1.1v */
+
+	rotation = <180>;
+	orientation = <2>;
+
+	port {
+		cam_endpoint: endpoint {
+			clock-lanes = <0>;
```

Comment:
I don't see bus-type being listed as mandatory in the bindings ?

Reply:
As I said, I get lost with what is and isn't validated in by the bindings, but if not included and the driver is ever extended to cover C-PHY, then you get into the world of pain of trying to retain backwards compatibility with existing users. 

Currently it looks to have minimal impact as C-PHY and D-PHY are largely identical through `__v4l2_fwnode_endpoint_parse`, but that may not always be the case.

Reply:
> As I said, I get lost with what is and isn't validated in by the bindings, but if not included and the driver is ever extended to cover C-PHY, then you get into the world of pain of trying to retain backwards compatibility with existing users.

True, let's avoid that, even if it's not an issue for validation it will make supporting C-PHY painless (or less painful)

> 
> Currently it looks to have minimal impact as C-PHY and D-PHY are largely identical through `__v4l2_fwnode_endpoint_parse`, but that may not always be the case.